egs/mini_an4/asr1/run.sh

```diff
@@ -11,6 +11,7 @@ backend=pytorch
 stage=-1       # start from -1 if you need to start from data download
 stop_stage=100
 ngpu=0         # number of gpus ("0" uses cpu, otherwise use gpu)
+decode_ngpu=0
```

Comment:
Probably we should do it for all recipes. Let's do it with the other PR.

Reply:
I agree